src/test/incomparable.rs

```diff
@@ -342,7 +342,7 @@ fn items() -> Result<()> {
 		quote! {
 			#[derive_where(PartialEq, PartialOrd)]
 			#[derive_where(incomparable)]
-			struct Test{}
```

Comment:
Interesting? Does it only work with unit struct and not with empty structs? What about empty tuple structs?

Reply:
it does work with all, probably could add test for them, but the generated code is actually identical.

Reply:
No I think that's fine, we can test it in integration tests.